### PR TITLE
Added UK Government coronavirus API

### PR DESCRIPTION
**Pull request by brychanthomas**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] Your additions are ordered alphabetically
- [x] Your submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column should be padded with one space on either side
- [x] You have searched the repository for any relevant issues or pull requests
- [x] Any category you are creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Review by 53jk1 — APPROVED** [review]

**Comment by marekdano** [comment]
Thanks for the contribution @brychanthomas!
Can you please resolve the conflict?

**Comment by brychanthomas** [comment]
I have resolved the conflict, but the checks are failing because [the link](https://coronavirus.data.gov.uk/developers-guide) returns a 404 status. However, the link still works - I assume there is a bug with the government's website.

**Comment by marekdano** [comment]
Thanks for resolving the conflict @brychanthomas!

Can I ask you to replace your link `https://coronavirus.data.gov.uk/developers-guide` with `https://coronavirus.data.gov.uk/details/developers-guide`? I can see that the link is redirected.
And the best is also to [squash](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit) all your commits.

File: README.md [hunk 1]
```diff
@@ -556,6 +556,7 @@ API | Description | Auth | HTTPS | CORS |
 ### Health
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
+| [Coronavirus in the UK](https://coronavirus.data.gov.uk/developers-guide) | UK Government coronavirus data, including deaths and cases by region | No | Yes | Unknown |
```
Comment (marekdano) [inline README.md:559]: Can I ask you to replace your link https://coronavirus.data.gov.uk/developers-guide with https://coronavirus.data.gov.uk/details/developers-guide? I can see that the link is redirected.

**Review by marekdano — CHANGES_REQUESTED** [review]
Thanks for resolving the conflict @brychanthomas!
I left a comment.

Can I ask you also to squash all your commits?

**Comment by brychanthomas** [comment]
Ah, I didn't notice the redirect! I have fixed the link and squashed it to one commit.

**Review by marekdano — APPROVED** [review]

**Pull request merged**